test/main_test.go

```diff
@@ -281,3 +290,76 @@ func deleteClusterWideTestOperators() {
 		}
 	}
 }
+
+func monitorOlmNamespace() {
+	monitorNamespace(config.GetOlmNamespace())
+}
+
+func monitorKogitoOperatorNamespace() {
+	monitorNamespace(kogitoClusterWideNamespace)
+}
+
+func monitorNamespace(namespace string) {
+	go func() {
+		err := framework.StartPodLogCollector(namespace)
+		if err != nil {
+			framework.GetLogger(namespace).Errorf("Error starting log collector for namespace %s: %v", namespace, err)
+		}
+	}()
+}
+
+func stopOlmNamespaceMonitoring() {
+	stopNamespaceMonitoring(config.GetOlmNamespace())
+}
+
+func stopKogitoOperatorNamespaceMonitoring() {
+	stopNamespaceMonitoring(kogitoClusterWideNamespace)
+}
+
+func stopNamespaceMonitoring(namespace string) {
```

Comment:
same for that one in framework package ?

Reply:
Not sure if it is worth it. This way it describes well the steps taken. Similar code is used in data.go too, though there are more steps defined.

Reply:
oki

Reply:
in that case, I would keep the `func monitorNamespace(namespace string) {`